src/models/input.ts

```diff
@@ -500,6 +500,29 @@ class InputModel {
         this.giveFocus();
     }
 
+    shouldRenderAuxViewKeybindings(view: InputAuxViewType): boolean {
```

Comment:
Here's the function that I implemented that contains logic for deciding when we should render keybindings